assets/simulation/px4_hitl_overview.svg

```diff
@@ -885,7 +885,7 @@
          id="tspan2761"
          x="-83.335098"
          y="147.11555"
-         style="font-size:3.88055563px;writing-mode:lr-tb;stroke-width:0.26458335px">simulator_mavlink.cpp</tspan></text>
+         style="font-size:3.88055563px;writing-mode:lr-tb;stroke-width:0.26458335px">mavlink_receiver.cpp</tspan></text>
```

Comment:
I find this a bit confusing: `mavlink_main.cpp` and `mavlink_receiver.cpp` are in different blocks, but it's actually the same module (and there are more source files involved for the communication). Should this not be a single block?

Also I noticed from the diagram and the text `QGroundControl is connected to the physical hardware over USB and acts as a gateway to forward data between the simulator running on a development computer, PX4, and any offboard API.`: this does not apply when jMAVSim is used for HITL: in that case, jMAVSim acts as gateway, connects to UART and forwards MAVLink so that QGC can connect (QGC is optional in that case).